package.json

```diff
@@ -13,15 +13,16 @@
   "license": "MIT",
   "type": "module",
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "website"
```

Comment:
Would you mind making another top level dir for `internal-packages/`, and adding `internal-packages/website` there? That would allow us to still use `internal-packages/*` for the `workspaces` and leaves us space to add internal test apps and whatnot (which I'm sure we'll want).

Reply:
done